### PR TITLE
feat: migrate grouped series config to mantine

### PR DESCRIPTION
**Pull request by magnew**
Closes: #6554 

### Description:

Migrates the top-level group series config to Mantine

### CODE REVIEW

File: packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx [hunk 1]
```diff
@@ -303,9 +309,9 @@ const GroupedSeriesConfiguration: FC<GroupedSeriesConfigurationProps> = ({
                             });
                         }}
                     />
-                </FormGroup>
+                </Stack>
             )}
-            <GroupSeriesWrapper>
+            <Box bg="gray.1" p={5} sx={{ borderRadius: 3 }}>
```
Comment (IrakliJani) [inline packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx:314]: is there a chance we can use `xxs` (4) for example?

also with borderRadius: yuo can use ```sx={(theme) => ({ borderRadius: theme.radius.sm })}` for example

File: packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx [hunk 2]
```diff
@@ -353,7 +359,8 @@ const GroupedSeriesConfiguration: FC<GroupedSeriesConfigurationProps> = ({
                                                         ref={innerRef}
                                                         {...draggableProps}
                                                     >
-                                                        <GroupedSeriesConfigWrapper
+                                                        <Box
+                                                            mb={5}
```
Comment (IrakliJani) [inline packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx:363]: same here, can we avoid 5 and stick with theme

File: packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/Series.styles.ts [hunk 3]
```diff
@@ -1,39 +1,5 @@
...
-    display: flex;
-    flex: 1;
-    gap: 0.714em;
-    justify-content: space-between;
-    margin-bottom: 0.714em;
-`;
+import { Colors } from '@blueprintjs/core';
+import styled from 'styled-components';
 
 export const SeriesBlock = styled.div`
```
Comment (IrakliJani) [inline packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/Series.styles.ts:4]: this is the only one that's left. can we do it here or is it going to be done somewhere else?

File: packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx (hunk 1 above)
Comment (magnew) [reply packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx:314]: Sure. I'll be a bit more careful with these.

File: packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx (hunk 2 above)
Comment (magnew) [reply packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx:363]: 👍

File: packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/Series.styles.ts (hunk 3 above)
Comment (magnew) [reply packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/Series.styles.ts:4]: Just posted the final review with the final cleanup.

Comment (IrakliJani) [reply packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/Series.styles.ts:4]: sweet!

**Review by IrakliJani — APPROVED** [review]
🚀

**Pull request merged**